app/src/main/java/org/desarrolladorslp/teckersapp/model/User.kt

```diff
@@ -0,0 +1,23 @@
+package org.desarrolladorslp.teckersapp.model
+import android.net.Uri
+
+class User(var Username: String?,var givenNameUser:String?, var familyNameUser:String?,var emailUser:String?,var idUser:String?, var photoUrlUser:Uri?) {
+
+    var nameUser:String? = null
+    var givenName:String? = null
+    var familyName:String? = null
+    var email: String? = null
+    var id: String? = null
+    var photoUrl: Uri? = null
```

Comment:
All these variables are unnecessary since they are declared in the constructor above.